docs/dns-usage.rst

```diff
@@ -0,0 +1,184 @@
+Using the API
+=============
+
+Authorization / Configuration
```

Comment:
1847126 reworks this section, including a link to the common doc (which looks kind of `datastore`-specific, BTW).